### DISCUSSION TITLE
Option to revert the old UI

### DISCUSSION THREAD

**Discussion by jeonghanlee** [jgraph/drawio#3215] [General]
Hi,
Is there any other way to configure to use the old UI? 
The new UI looks pretty only from my ugly view. I am spending more time now understanding the new UI. It isn't a very comfortable experience.

**Comment by Wladefant** [comment]
Well yes, there are like 10 different uis you can try

**Reply by jeonghanlee** [reply]
Oh, thanks. I found the way to switch UIs.

**Reply by annie-maria** [reply]
Can you share how? I just want to fix the trays as they were before.

**Reply by jeonghanlee** [reply]
@annie-maria, you can switch them through the menu. It is a bit tricky to understand their menu name at first sight.

```
Extra > Theme >  Default or Automatic
```

**Reply by Wladefant** [reply]
That is completely different in different menus

**Reply by annie-maria** [reply]
Thank you!

**Reply by davidjgraph** [reply]
https://www.diagrams.net/blog/modern-mode-diagrams
